Lib/fontTools/misc/plistlib.py

```diff
@@ -1,13 +1,14 @@
+import collections.abc
 import sys
 import re
+from typing import Any, Callable, Dict, List, NoReturn, Optional, Sequence, Tuple, Union, overload
+from typing import Any, Callable, Dict, List, Mapping, NoReturn, Optional, Sequence, Tuple, Type, TypeVar, Union, overload, IO
```

Comment:
(perhaps the PR is unfinished but) I see two almost duplicated `from typing import *` lines above.

Reply:
Yeah, I'll clean up later. Still fudging about between random support requests.